web/test/pages/onboarding/onboarding-owning.test.tsx

```diff
@@ -68,17 +68,15 @@ describe("onboarding - owning a business", () => {
       expect(screen.getByLabelText("Sector")).toBeInTheDocument();
     });
 
-    it("prevents user from moving after Step 1 if you have not entered a sector", async () => {
-      const userData = generateTestUserData({ sectorId: undefined });
-      useMockRouter({ isReady: true, query: { page: "1" } });
-      const { page } = renderPage({ userData });
-      page.clickNext();
+    it("does not allow OWNING user persona to move past Step 1 if user has not entered a sector", async () => {
+      const { page } = renderPage({ userData: undefined });
+      page.chooseRadio("business-persona-owning");
+      fireEvent.click(screen.getByTestId("next"));
       await waitFor(() => {
-        expect(screen.getByTestId("step-1")).toBeInTheDocument();
+        expect(
+          screen.getByText(Config.profileDefaults.fields.sectorId.default.errorTextRequired)
+        ).toBeInTheDocument();
       });
-      expect(
-        screen.getByText(Config.profileDefaults.fields.sectorId.default.errorTextRequired)
-      ).toBeInTheDocument();
       expect(screen.getByTestId("banner-alert-REQUIRED_REVIEW_INFO_BELOW")).toBeInTheDocument();
```

Comment:
[dust] might be cool to some how check that we're still on the page at the end (right now we're only checking that there is an error so their might be a weird case where we move pages and show an error) but I get it if you think this is overkill.

Reply:
oh that's a good idea

Reply:
@adidner, can you share more about what you mean when you write, "weird case where we move pages and show an error"? There are no other required fields for the OWNING persona in onboarding, and once the user selects the OWNING person and a sector, they are directed to the dashboard.

Reply:
@AnJuHyppolite I think @adidner  is saying that we could have another expect statement like: 
`expect(screen.getByTestId("step-2")).toBeInTheDocument();`
to confirm that the error stops the user from navigating forward

Reply:
Yeah, 

So imagine a situation where the user, selects oscar, doesn't enter a sector, hits the next button and the app goes to the next onboarding page and then pulls up the error. 

This would be wrong as we don't want them to move past the first page without a sector but the condition I described will get your test to pass. 

The test title says they cannot move past page 1 but you don't actually check that we're still on page 1, you're just checking for an error popup, so I'm just saying it might be cool to have 1 more expect to confirm we are still on page 1. 

Does that help clarify things?